Dockerfile

```diff
@@ -26,8 +29,18 @@ RUN  curl -sSL https://install.python-poetry.org | python3 -
 
 WORKDIR /app/
 COPY ./poetry.lock ./pyproject.toml ./
+COPY ./package-lock.json ./package.json .nvmrc ./
+COPY ./webpack.config.js ./webpack.config.prod.js ./
+COPY ./tsconfig.json ./eslint.config.mjs ./
+COPY ./jest.config.ts ./
+
+RUN mkdir -p $NVM_DIR && curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v${NVM_VERSION}/install.sh | bash
+
+RUN . ~/.bashrc && nvm install && nvm use
 
 RUN poetry install --without dev,test
+RUN . "$NVM_DIR/nvm.sh" && npm install
+ENV PATH="$NVM_DIR/versions/node/v20.16.0/bin:$PATH"
```

Comment:
Can't we put the node version in a variable instead of hardcoding it this way?

Reply:
No, because it is read from .nvmrc. i could not get it to work in any other way then hardcoding it.

Reply:
I already tried for 3 hours. so prefer not spending another 10 to try to get it to work